### DISCUSSION TITLE
Great library thank you!

### DISCUSSION THREAD

**Discussion by brianmontanamvmnt** [valendres/playwright-msw#59] [Show and tell]
Thanks creating the library, I was about to setup a hoisting approach myself but you addressed my big concerns for scenario testing svelte 🙏 

If you need any additional help feel free to reach out, really appreciate the library!

**Comment by valendres** [comment]
It's my pleasure @brianmontanamvmnt. I'm glad you find it useful 🙂

I'll definitely reach out if needed!
